README.md

```diff
@@ -20,4 +20,32 @@ This repository is configured for Nat Pryce's [_adr-tools_](https://github.com/n
 
 API definitions are in OAS (Swagger 2.0).
 
-# The Digital Citizenship Initiative
+## Deploy Azure PaaS
+
+This repository contains scripts to deploy needed PaaS on Azure cloud.
+
+### Prerequisites
+
+- [Git](https://git-scm.com/)
+- [NodeJS](https://nodejs.org/it/)
+- [Terraform](https://terraform.io) 
```

Comment:
should specify version, since TF config may change from one version to another

Reply:
ok